ios/Capacitor/Capacitor/CAPUNUserNotificationCenterDelegate.swift

```diff
@@ -62,11 +62,9 @@ public class CAPUNUserNotificationCenterDelegate : NSObject, UNUserNotificationC
       }
     }
 
-    if (self.bridge?.isAppActive())! {
-      completionHandler([.badge, .sound, .alert])
-    } else {
-      completionHandler([.badge, .sound])
-    }
+    // This method is only run when the app is in the foreground, therefore
+    // always suppress the alert notification
+    completionHandler([.badge, .sound])
```

Comment:
This delegate is also used for local notifications, and I think it's desired to show them even when in foreground, so you should only remove the alert for push notifications.

Reply:
I moved this to a different pull here and addressed your feedback: https://github.com/ionic-team/capacitor/pull/1223